service/worker/deletenamespace/fx.go

```diff
@@ -54,20 +65,36 @@ type (
 	}
 	componentParams struct {
 		fx.In
-		VisibilityManager manager.VisibilityManager
-		MetadataManager   persistence.MetadataManager
-		HistoryClient     resource.HistoryClient
-		MetricsHandler    metrics.Handler
-		Logger            log.Logger
+		ActivityWorkerConfig activityWorkerConfig
+		VisibilityManager    manager.VisibilityManager
+		MetadataManager      persistence.MetadataManager
+		HistoryClient        resource.HistoryClient
+		MetricsHandler       metrics.Handler
+		Logger               log.Logger
 	}
 )
 
-var Module = workercommon.AnnotateWorkerComponentProvider(newComponent)
+var Module = fx.Options(
+	fx.Provide(newActivityWorkerConfig),
+	workercommon.AnnotateWorkerComponentProvider(newComponent),
+)
+
+func newActivityWorkerConfig(
+	dc *dynamicconfig.Collection,
+) activityWorkerConfig {
+	return activityWorkerConfig{
+		maxConcurrentActivityExecutionSize: dc.GetIntProperty(dynamicconfig.WorkerDeleteNamespaceMaxConcurrentActivityExecutionSize, 0),
+		taskQueueActivitiesPerSecond:       dc.GetFloat64Property(dynamicconfig.WorkerDeleteNamespaceTaskQueueActivitiesPerSecond, 1000),
```

Comment:
I think we can use 0 as rely on SDK default here. we only care about the activity concurrent execution size.